sftp/1.3/modules/ROOT/edits/sftp-list.adoc

```diff
@@ -0,0 +1,243 @@
+	xmlns:ftps="http://www.mulesoft.org/schema/mule/ftps"
+	xmlns:sftp="http://www.mulesoft.org/schema/mule/sftp"
+	xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/sftp http://www.mulesoft.org/schema/mule/sftp/current/mule-sftp.xsd
+http://www.mulesoft.org/schema/mule/ftps http://www.mulesoft.org/schema/mule/ftps/current/mule-ftps.xsd
+http://www.mulesoft.org/schema/mule/ee/core http://www.mulesoft.org/schema/mule/ee/core/current/mule-ee.xsd
+http://www.mulesoft.org/schema/mule/os http://www.mulesoft.org/schema/mule/os/current/mule-os.xsd">
+
+	<sftp:config name="SFTP_Config" doc:name="SFTP Config" doc:id="41238701-97e6-4c85-a073-7fd2edf93416" >
+		<sftp:connection host="localhost" port="2222" username="bob" password="pass"/>
+	</sftp:config>
+
+	<os:object-store name="Object_store" doc:name="Object store" doc:id="890b7561-32db-4d65-bb67-9fef05c6f6fe" />
+
+	<flow name="poll">
+	  <scheduler>
+	      <scheduling-strategy>
+	          <fixed-frequency frequency="1000"/>
+	      </scheduling-strategy>
+	  </scheduler>
+
+	  <sftp:list directoryPath="/config/dropFolder" config-ref="SFTP_Config"/>
+
+	  <foreach>
+	      <flow-ref name="processFile" />
+	      <sftp:delete path="#[attributes.path]" config-ref="SFTP_Config"/>
+	  </foreach>
+	</flow>
+
+	  <flow name="processFile" maxConcurrency="1">
+			<ee:transform doc:name="Transform Message" doc:id="9f39c6d4-9356-427f-b497-dd2a9099939c" >
+				<ee:message >
+					<ee:set-payload ><![CDATA[%dw 2.0
+						output application/json
+						---
+						characters: payload.characters.*name map (
+						            (item, index) -> {name: item}
+						         )]]></ee:set-payload>
+				</ee:message>
+			</ee:transform>
+
+			<logger level="ERROR" message="#[payload]" />
+			<os:store doc:name="Store" objectStore="Object_store" key="#['test-file-' ++ random() as String ++ '.json']"/>
+	  </flow>
+
+</mule>
+
+----
+
+== Match Filter
+
+When listing files, use the File Matching Rules parameter that accepts files that match the specified criteria. This parameter defines the possible attributes to either accept or reject a file.
+These attributes are optional and ignored if you do not provide values for them. Use an `AND` operator to combine individual attributes.
+
+To configure the parameter in Studio, set the *File Matching Rules* parameter to `Edit inline` and complete the desired attributes:
+
+* *Timestamp since* +
+Files created before this date are rejected.
+* *Timestamp until* +
+Files created after this date are rejected.
+* *Not updated in the last* +
+Minimum time that should pass since a file was last updated to not be rejected. This attribute works in tandem with *Time unit*.
```

Comment:
Is this correct?
```suggestion
Maximum time that can pass between updates before a file is rejected. This attribute works in tandem with *Time unit*.
```

Reply:
Oh, now I see the next one. Okay, I don't understand this. I am assuming that after a certain time passes between updates, an old, not updated file becomes invalid and is rejected: is this true?
There is awkwardness in "time that should pass from when a file was last updated to not be rejected" that I can't figure out how to reword for more clarity: can you check in with a peer writer on this?